docs/it/extend/update-b10.md

```diff
@@ -0,0 +1,51 @@
+# Updating For Beta 10
```

Comment:
If these aren't gonna be translated (which is fine, they're outdated and not essential) they should be removed.